wormhole-connect/src/views/v2/Bridge/AssetPicker/TokenItem.tsx

```diff
@@ -13,6 +13,9 @@ import TokenIcon from 'icons/TokenIcons';
 import { ChainConfig, TokenConfig } from 'config/types';
 import config from 'config';
 
+import { Chain } from '@wormhole-foundation/sdk';
```

Comment:
Nitpick: Maybe to add the `type` operator?

Reply:
changed